src/qibolab/pulses.py

```diff
@@ -21,6 +21,7 @@ class PulseType(Enum):
     READOUT = "ro"
     DRIVE = "qd"
     FLUX = "qf"
+    COUPLERFLUX = "cf"
```

Comment:
They are almost the same. I found it was simpler for me when passing them in my driver to check if they were `COUPLERFLUX` pulses to assign them in the right place. The main difference would be that they have something in `pulse.coupler` directly instead of having to check `FLUX` pulses `pulse.qubit` if it's a `Qubit` or `Coupler` what you are getting. It seemed strange or making things harder to read to check that.